packages/eas-cli/src/commands/update/index.ts

```diff
@@ -211,7 +211,7 @@ export default class UpdatePublish extends EasCommand {
       projectId,
     });
 
-    const { exp } = await getDynamicProjectConfigAsync({});
+    const { exp } = await getDynamicProjectConfigAsync({ isPublicConfig: true });
```

Comment:
I was curious about the blame, looks like the blame rev was 8d0d34c497459c9bd36cb3ea89cb433d5e5dd52b. Looking at the PR, this would have been super hard to catch in code review (neither author nor reviewer error). I'd argue that the default for this is wrong. It should default closed, require override to open; i.e. default to public.

My guess is that there are probably a lot of other places we've made this error due to this API.